src/pseudopeople/noise_entities.py

```diff
@@ -20,7 +20,16 @@ class __NoiseTypes(NamedTuple):
         "do_not_respond", noise_functions.apply_do_not_respond
     )
     omit_row: RowNoiseType = RowNoiseType("omit_row", noise_functions.omit_rows)
-    # duplicate_row: RowNoiseType = RowNoiseType("duplicate_row", noise_functions.duplicate_rows)
+    duplicate_with_guardian: RowNoiseType = RowNoiseType(
+        "duplicate_with_guardian",
+        noise_function=noise_functions.duplicate_with_guardian,
+        row_probability={
```

Comment:
I still think it would be cleaner to treat these as `additional_parameters` like the column noise types with the `row_probability` set to 1.0 for duplicate_wth_guardian

Reply:
If we do that we then will have a row probability that means nothing and users may change it thinking it is relevant for noising this type.

Reply:
So floats do work so I guess it is fine.